spinnman/processes/abstract_process.py

```diff
@@ -11,37 +12,45 @@ class AbstractProcess(object):
     """ An abstract process for talking to SpiNNaker efficiently
     """
     __slots__ = [
-        "_error_request",
-        "_exception",
-        "_traceback"]
+        "_error_requests",
+        "_exceptions",
+        "_tracebacks"]
 
     def __init__(self):
-        self._exception = None
-        self._traceback = None
-        self._error_request = None
+        self._exceptions = list()
+        self._tracebacks = list()
+        self._error_requests = list()
 
     def _receive_error(self, request, exception, tb):
-        self._error_request = request
-        self._exception = exception
-        self._traceback = tb
+        self._error_requests.append(request)
+        self._exceptions.append(exception)
+        self._tracebacks.append(tb)
 
     def is_error(self):
-        return self._exception is not None
+        return self._exceptions is not None
```

Comment:
Well that's plain wrong. The `_exceptions` is never `None`, so there's always an error.